src/libmongoc/src/mongoc/mongoc-util-private.h

```diff
@@ -162,6 +162,33 @@ _mongoc_document_is_pipeline (const bson_t *document);
 char *
 _mongoc_getenv (const char *name);
 
+/*
+ *--------------------------------------------------------------------------
+ *
+ * _mongoc_setenv --
+ *
+ *       Set or overwrite the value of an environment variable.
+ *
+ * Returns:
+ *       False if setting the variable was unsuccessful.
+ *
+ *--------------------------------------------------------------------------
+ */
+bool
+_mongoc_setenv (const char *name, const char *value);
+
+BSON_EXPORT (void)
+bson_copy_to_including_noinit (const bson_t *src,
+                               bson_t *dst,
+                               const char *first_include,
+                               ...) BSON_GNUC_NULL_TERMINATED;
+
+BSON_EXPORT (void)
```

Comment:
```suggestion
void
```

Remove `BSON_EXPORT` for private symbols.